remote_attestation/rust/src/crypto.rs

```diff
@@ -234,7 +235,7 @@ impl KeyNegotiator {
             &agreement::UnparsedPublicKey::new(KEY_AGREEMENT_ALGORITHM, peer_public_key),
             anyhow!("Couldn't derive session keys"),
             |key_material| {
-                let key_material = key_material.try_into().context(format!(
+                let key_material = key_material.try_into().context(alloc::format!(
```

Comment:
I would suggest just re-use all these things from alloc, instead of qualifying them everywhere. In fact I'm surprised they are not already in some prelude. Maybe we should build our own prelude? (see https://doc.rust-lang.org/core/prelude/v1/index.html)

Reply:
It seems that there used to be an experimental alloc prelude, but it no longer exists. Importing custom preludes does not seem to be encouraged (https://doc.rust-lang.org/beta/unstable-book/language-features/prelude-import.html), so I added use statements as needed.

Reply:
I don't see anything that discourages custom preludes, a lot of libraries do that. The feature to bake a prelude in the compiler is another story, I don't recommend we use that, just `use oak_uefi::prelude::*` basically.

Reply:
Do you mean that we have to add the use statement to all modules that use it? Or is there another way (apart from `prelude-import`) to use it for the entire crate?

Reply:
Yes we would need to import it. e.g. https://docs.rs/futures/latest/futures/prelude/index.html

Reply:
Created https://github.com/project-oak/oak/issues/2564